Parsers/Summarize the thread so far.js

```diff
@@ -27,7 +27,7 @@ flags:gmi
     if (thread_ts) {
         chatGr.addEncodedQuery('thread_ts=' + thread_ts + '^ORts=' + thread_ts + '^textNOT LIKE!catchmeup^textNOT LIKE!summary^textNOT LIKE!catchup');
     } else {
-        chatGr.addEncodedQuery('channel=' + current.getValue('channel') + '^sys_created_onONLast 30 minutes@javascript:gs.beginningOfLast30Minutes()@javascript:gs.endOfLast30Minutes()^thread_ts=NULL^textNOT LIKE!catchmeup^textNOT LIKE!summary^textNOT LIKE!catchup');
+        chatGr.addEncodedQuery('channel=' + current.getValue('channel') + '^sys_created_onONLast hour@javascript:gs.beginningOfLastHour()@javascript:gs.endOfLastHour()^thread_ts=NULL^textNOT LIKE!catchmeup^textNOT LIKE!summary^textNOT LIKE!catchup');
```

Comment:
sys_created_onRELATIVEGT@minute@ago@60 should work instead

Reply:
could let it take a param for up to some number of messages to prevent too much token usage. Or even summarizing just on a count of messages could work. 🤷

Reply:
changed to relative query. I think i'd rather let someone do the param thing as a feature add lol